src/AgOledDisplay.h

```diff
@@ -40,6 +40,8 @@ class OledDisplay : public PrintLog {
   void showNewFirmwareUpdating(String percent);
```

Comment:
propose to rename to
showFormwareUpdateProgress(int percent)

Reply:
Ok